.github/workflows/maven.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Checkout core
-      run: git clone --depth=50 --branch=develop https://github.com/informatici/openhospital-core.git openhospital-core
+      run: git clone --depth=50 --branch=OP-162_ResultSet_Pagination_prototype https://github.com/informatici/openhospital-core.git openhospital-core
```

Comment:
doesn't this need to be set back to `develop` when this PR is committed?

Reply:
yes, updated the description, thanks

Reply:
What I meant was shouldn't the `branch` value just be `develop`?   This workflow is for CI jobs for the whole repo, no?
Maybe I don't understand.